doc/api-extensions.md

```diff
@@ -1208,3 +1208,6 @@ When creating the network if these settings are unspecified, and an equivalent I
 the subnet, then the appropriate NAT setting will added set to `true`.
 
 If the setting is missing then the value is taken as `false`.
+
+## tpm\_device\_type
```

Comment:
Can you move that change to the other API extension commit?

We usually do a commit like `api: Add tpm_device_type extension` which contains both the `shared/version/api.go` and `doc/api-extensions.md` bits.